changelog/std-traits-hasStaticMember.dd

```diff
@@ -0,0 +1,13 @@
+Added `std.traits.hasStaticMember` to check whether a symbol is a static member of a type.
```

Comment:
https://wiki.dlang.org/Guidelines_for_maintainers#Write_access_to_PRs

It's enabled here:

![image](https://cloud.githubusercontent.com/assets/4370550/23580654/f83f885a-0105-11e7-8e32-535d1ebe4f1b.png)

Anyways we put this into the reminder message of the DLang-Bot:
https://github.com/dlang-bots/dlang-bot/pull/44